modules/luci-base/htdocs/luci-static/resources/ui.js

```diff
@@ -610,6 +610,18 @@ var UICheckbox = UIElement.extend(/** @lends LuCI.ui.Checkbox.prototype */ {
 
 		frameEl.appendChild(E('label', { 'for': id }));
 
+		var tooltip = this.options.tooltip(this.options.name);
+		if (typeof(tooltip) === 'string') {
+			frameEl.appendChild(
+				E('label', { 'class': 'cbi-tooltip-container' },[
+					"⚠️",
```

Comment:
Not sure if we should hardcode this emoji as tooltip element / text. Maybe introduce another property `tooltiptitle` or similar which allows overriding it.

Reply:
Made changeable with commit https://github.com/openwrt/luci/pull/4667/commits/8a3fbcb18db5ae99a35e0de289bd969bb3891a97